wikihow2zim/scraper.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+options = [
+    "debug",
+    "language",
+    "low_quality",
+    "no_external_links",
+    "s3_url_with_credentials",
+]
+
+class wikihow2zim:
+    def __init__(self, **kwargs):
+        print("kwargs", kwargs)
+
+        for option in options:
+            if option not in kwargs:
+                raise ValueError(f"Missing parameter `{option}`")
+
+        self.debug = kwargs.get("debug")
```

Comment:
For simple assignments, you can factorize this code to save several almost identical lines.